PixaPicker/PixaCollectionViewCell.swift

```diff
@@ -11,6 +11,17 @@ import UIKit
 
 class PixaCollectionViewCell : UICollectionViewCell {
     @IBOutlet weak var cellImage: UIImageView!
+    @IBOutlet weak var favoriteButton: UIButton!
+    @IBOutlet weak var saveButton: UIButton!
+    var delegate: PixaSaveButtonDelegate?
```

Comment:
`delegate` variables always need to be labeled `weak`, or they might end up in what's known as a "retain cycle".

More info here: https://medium.com/@vinodhswamy/strong-cycle-retain-cycle-in-swift-f452f07518b2